test/convert_test.go

```diff
@@ -54,30 +53,46 @@ func TestConvert(t *testing.T) {
 		assetsByType[a.Type] = append(assetsByType[a.Type], a)
 	}
 
+	jsonFixtures := make(map[string][]byte)
+
+	matches, _ := filepath.Glob(filepath.Join(jsonGenerateDir, "*.json"))
+	for _, fixturePath := range matches {
+		fixtureFileName := strings.TrimPrefix(fixturePath, jsonGenerateDir+"/")
+		fixtureName := strings.TrimSuffix(fixtureFileName, ".json")
+
+		fixtureData, err := ioutil.ReadFile(fixturePath)
+		if err != nil {
+			t.Fatalf("Reading: %v", err)
+		}
+
+		jsonFixtures[fixtureName] = fixtureData
+	}
+
 	t.Run("Disk", func(t *testing.T) {
-		requireEqualJSONValues(t,
-			// Expected:
-			data.Disk,
-			// Received:
+		requireEqualJSON(t,
```

Comment:
since the test cases now only differ in fixture name and asset type, can we change this into a table driven test instead of having to explicitly call t.Run() on each case?

Reply:
Good idea, implemented in 3c1fd17d3e678f24e9a4d21a98e7755efad7d5e4